sosivio/README.md

```diff
@@ -0,0 +1,23 @@
+# Sosivio
+ 
+## Overview
+ 
+Integrate Sosivio with DataDog to get Your Sosivio alerts and notifications sent directly to DataDog!
```

Comment:
It may be helpful to add a bit more detail about what Sosivio does, and the types of data being sent into Datadog